Makefile

```diff
@@ -111,17 +111,18 @@ unit-test:
 
 .PHONY: ci-unit-test
 ci-unit-test: $(GO_JUNIT_REPORT) $(GOVERALLS)
+	go test -short -v ./... | go-junit-report;
```

Comment:
This looks like a bad merge. `ci-unit-test` should still report coverage when possible.

Reply:
This was purposeful. I wanted to get coverage reports from the integration tests as they should have greater coverage. That being said, I think we can have multiple hooks for parallel builds with coveralls, so having this should be fine

Reply:
I'm not sure I understand. This target is for unit tests.